package.json

```diff
@@ -35,6 +35,14 @@
 			"type": "object",
 			"title": "Beautify config",
 			"properties": {
+				"beautify.config": {
+					"type": [
+						"string",
+						"null"
+					],
+					"default": null,
+					"description": "A path to file containing the configuration options for js-beautify. If the file exists it overrides any .jsbeautify file"
```

Comment:
Again, I wouldn't set this as the default. Perhaps target tree first, then this setting.